src/blocks/accordion/group/save.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies.
+ */
+const { InnerBlocks } = wp.blockEditor;
+
+const Save = ({
+	attributes,
+	className
+}) => {
+	return (
+		<div
+			id={ attributes.id }
+			className={ classnames(
+				className,
+				{
+					[`is-${ attributes.gap }-gap`]: attributes.gap
+				}
+			) }
```

Comment:
`classNames` is undefined

Reply:
I've just kept it like this as it's the same for the rest of the blocks. When we merge TS, we can change this.